spinnman/spalloc/spalloc_client.py

```diff
@@ -194,6 +206,31 @@ def create_job_board(
             "keepalive-interval": f"PT{int(keepalive)}S"
         }, machine_name)
 
+    @overrides(AbstractSpallocClient.create_job_rect_at_board)
+    def create_job_rect_at_board(
```

Comment:
Consider self, width, height, *, triad=Non

This forces all callers to name the optional param